src/bin/pg_dump/dump_babel_utils.c

```diff
@@ -166,6 +166,43 @@ dumpBabelGUCs(Archive *fout)
 	destroyPQExpBuffer(qry);
 }
 
+void
+dumpBabelMigrationModeCheck(Archive *fout)
```

Comment:
will it be applicable to MVU as well? We are already blocking it CP level but still will this check be dumped during APG MVU